LiteCore/tests/CMakeLists.txt

```diff
@@ -102,3 +100,5 @@ target_link_libraries(
     Support
     ${LITECORE_CRYPTO_LIB}
 )
+
```

Comment:
Why is this necessary?  ... or, perhaps, just desirable?

Reply:
It is so that things like ICU and such show up _after_ everything else to satisfy GCC's linker which only supplies symbols to satisfy previously undefined ones in the linker input.  Without this things like deflate, and ICU functions show up as "undefined reference" and the link fails.  This was addressed in a more robust way in 2.8+.

Reply:
Got it.  ... and the 2.8+ way is to big a change to put here?

Reply:
It could be done but it didn't seem worth it because the 2.8 change was basically a CMake project overhaul.  This change accomplishes what is needed while keeping changes to a minimum.

Reply:
Yep.  Got it.  Approved.